moveit_planners/trajopt/trajopt_interface_plugin_description.xml

```diff
@@ -0,0 +1,6 @@
+<library path="libmoveit_trajopt_planner_plugin">
+  <class name="trajopt_interface/TrajOptPlanner" type="trajopt_interface::TrajOptPlannerManager" base_class_type="planning_interface::PlannerManager">
+    <description>
```

Comment:
Add desc of what Trajopt is (you can copy this from package.xml if you want)

Reply:
done